Source/objects.cpp

```diff
@@ -2905,18 +2905,12 @@ int ItemMiscIdIdx(int imiscid)
 
 void OperateShrine(int pnum, int i, int sType)
 {
-	int v1;
-	int v12;  // edx
-	int v21;  // eax
-	int v60;  // ebx
-	int j;    // edi
-	int v72;  // edi
-	int v88;  // ebx
-	int v107; // ST38_4
-	int v108; // ST34_4
-	int v133; // eax
+	int cnt;
+	int r, j;
+	unsigned long lv, t;
 	int xx, yy;
-	int min, max;
+	int v1, v2, v3, v4;

```

Comment:
Maybe name them:
```suggestion
	int str, mag, dex, vit;
```

Reply:
Yeah I had them named like that and then tried to match them with names from PSX. Not sure how hard I should try to match them after all, `v1`-`v4` is pretty stupid names although one of them seems to be featured in assert though.

Reply:
Arh didn't know they where from the PSX file, in that case lets keep them :)